libs/libcommon/src/libcommon/queue.py

```diff
@@ -372,18 +391,26 @@ def __exit__(
         return False
 
     @classmethod
-    def git_branch(cls, dataset: str, branch: str, owner: str, sleeps: Sequence[float] = _default_sleeps) -> "lock":
+    def git_branch(
+        cls,
+        dataset: str,
+        branch: str,
+        owner: str,
+        sleeps: Sequence[float] = _default_sleeps,
+        ttl: Optional[_TTL] = None,
+    ) -> "lock":
         """
         Lock a git branch of a dataset on the hub for read/write
 
         Args:
             dataset (`str`): the dataset repository
             branch (`str`): the branch to lock
             owner (`str`): the current job id that holds the lock
-            sleeps (`Sequence[float]`): the time in seconds to sleep between each attempt to acquire the lock
+            sleeps (`Sequence[float]`, optional): the time in seconds to sleep between each attempt to acquire the lock
+            ttl (`integer`, optional): either lock.TTL.LOCK_TTL_SECONDS_TO_START_JOB or lock.TTL.LOCK_TTL_SECONDS_TO_WRITE_ON_GIT_BRANCH
```

Comment:
why not hardcode `lock.TTL.LOCK_TTL_SECONDS_TO_WRITE_ON_GIT_BRANCH` and remove the `ttl` argument in this method?

Reply:
done :)